labonneboite/common/refresh_peam_token.py

```diff
@@ -0,0 +1,20 @@
+from flask import flash, url_for
+from flask_login import current_user
+
+from labonneboite.common.models.auth import TokenRefreshFailure
+
+
+def attempt_to_refresh_peam_token():
+    if current_user.is_authenticated:
+        try:
+            current_user.refresh_peam_access_token_if_needed()
+        except TokenRefreshFailure:
+            message = "Votre session PE Connect a expiré. Veuillez vous reconnecter."
+            flash(message, 'warning')
+            return {
```

Comment:
why not return just the URL?

there are only 2 cases right? 

1- token_has_expired + redirect_url
2- nothing

Reply:
J'ai commencé comme ça mais je trouvais ça illisible. Je trouve plus propre/lisible de renvoyer un dict qui donne deux infos, si oui ou non il y a besoin de redirect et dans le cas échéant l'url.

Reply:
Comment tu nommerais ta méthode avec ta solution?